Makefile

```diff
@@ -15,13 +15,26 @@ format:
 
 lint:
 	cargo fmt --all -- --check
-	cargo clippy -- -D warnings -A clippy::derive_partial_eq_without_eq
+	cargo clippy --tests -- -D warnings -A clippy::derive_partial_eq_without_eq
 
 build:
+	make fetch-thirdparty
```

Comment:
We should split `thirdparty/account-abstraction` and `thirdparty/bundler` and have the latter only on `make test`. I do not think any of the sc will be used in the main build.